scripts/example_NASA_FIRMS.py

```diff
@@ -0,0 +1,39 @@
+from pyronear_ds.datasets import NASAFIRMS
+
+
+def main(args):
+
+    nasa_firms = NASAFIRMS(args.nasa_firms, args.nasa_firms_type)
+    print(nasa_firms)
+
+
+def parse_args():
+    import argparse
+
+    parser = argparse.ArgumentParser(
+        description="Pyronear weather & wildfire history example",
```

Comment:
Is there weather data in NASAFIRMS? If not, we should adapt the script prompt here I think!

Reply:
done